kubernetes/kustomize/cornserve/base/role-bindings.yaml

```diff
@@ -0,0 +1,109 @@
+---
+# Gateway: CR access
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: gateway-cr-access
+  namespace: cornserve
+subjects:
+- kind: ServiceAccount
+  name: gateway-sa
+  namespace: cornserve
+roleRef:
+  kind: Role
+  name: cr-access-role
+  apiGroup: rbac.authorization.k8s.io
+
```

Comment:
This chunk should be separated out as `kubernetes/kustomize/cornserve/base/gateway/role-binding.yaml` and `role-binding.yaml` should be added to `kubernetes/kustomize/cornserve/base/gateway/kustomization.yaml`. The same applies to all other services and also for `roles.yaml`.